lib/firestore.js

```diff
@@ -169,11 +184,12 @@ export class FirestoreBackup {
       throw new Error('Unable to create backup path for Collection \'' + collection.id + '\': ' + error)
     }
 
-    return Promise.resolve(collection.get())
+    return this.Promise.resolve(collection.get())
       .then((documentSnapshots) => documentSnapshots.docs)
       .map((document) => {
         return this.backupDocument(document, backupPath + '/' + document.id, logPath + collection.id + '/')
       }, { concurrency: this.options.requestCountLimit })
+      .catch(err=>console.error(err))
```

Comment:
I was wondering why we aren't bubbling up our errors anymore in the Promise's return result

Reply:
Ah my mistake, you removed those in the latest commit!